source/grammar/tests/invalid/statements/switch.qasm

```diff
@@ -0,0 +1,5 @@
+switch () {}
+switch (i) { x $0 }
+switch (i) { case {} }
+switch (i) { case 1,, {} }
+switch (i) { default 0 {} }
```

Comment:
I can add one with defaults separated by commas, but having multiple separate `default` cases wouldn't be an error in this test, since this file is just for grammatical errors, not semantic ones.  I'll add a semantic test case to the AST test suite for multiple `default`, though.

Reply:
Extra invalid-grammar test added in edf15ed.